pages/docs/guides/webhooks/create-a-webhook-subscription/obtain-authorization.mdx

```diff
@@ -1,58 +0,0 @@
----
-layout: guides
-title: "Step 1: Obtain Authorization"
-guide:
-  step: 1
-meta:
-  title: Webhooks | Obtain Authorization
-  description: Obtain authorization webhooks for payments within your application by utilizing our open bank transfer API with no transaction fees.
----
```

Comment:
Looks like we lost this step somewhere. If we're purposefully removing it, we'll have to adjust the other steps to fix the broken chain